platform/security/idp/security-idp-client/src/main/java/org/codice/ddf/security/idp/client/AssertionConsumerService.java

```diff
@@ -113,13 +138,70 @@ public AssertionConsumerService(SimpleSign simpleSign, IdpMetadata metadata,
     }
 
     @POST
+    @Consumes({"*/*"})
     @Produces(MediaType.APPLICATION_FORM_URLENCODED)
     public Response postSamlResponse(@FormParam(SAML_RESPONSE) String encodedSamlResponse,
             @FormParam(RELAY_STATE) String relayState) {
 
         return processSamlResponse(decodeBase64(encodedSamlResponse), relayState);
     }
 
+    @POST
+    @Consumes({"text/xml", "application/soap+xml"})
+    public Response processSoapResponse(InputStream body, @Context HttpServletRequest request) {
+        try {
+            SOAPPart soapMessage = SamlProtocol.parseSoapMessage(IOUtils.toString(body));
+            String relayState = getRelayState(soapMessage);
+            org.opensaml.saml.saml2.core.Response samlpResponse = getSamlpResponse(soapMessage);
+            boolean validateResponse = validateResponse(samlpResponse);
+            if (validateResponse) {
```

Comment:
There should have already been something logged by the validator itself at this point.